app/build.gradle

```diff
@@ -220,6 +220,8 @@ dependencies {
     //Permissions management library for Jetpack Compose
     implementation "com.google.accompanist:accompanist-permissions:0.21.1-beta"
 
+    implementation  "androidx.compose.material3:material3:1.0.0-alpha06"
```

Comment:
What's this for?

Reply:
It is for the nested scroll behaviour in the Chat screen